bastionzero/service/targets/targets.go

```diff
@@ -125,7 +127,12 @@ type VirtualTarget struct {
 
 	// ProxyTargetID is the ID of the target that proxies connections made to
 	// this virtual target
+	// One of either ProxyTargetID or ProxyEnvironmentID must be specified when creating a target
```

Comment:
Also add this sentence in the comment above the `CreateDatabaseTargetRequest` and `CreateWebTargetRequest` structs

Reply:
[9594f75](https://github.com/bastionzero/bastionzero-sdk-go/pull/42/commits/9594f75b8b24ba30874e6bcbc2c6766d5a5fe9c7)